artseenapi/models/piece.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+class Piece(models.Model):
+    artist = models.ForeignKey("Artist", on_delete=models.CASCADE)
+    title = models.CharField(max_length=200)
+    subtitle = models.CharField(max_length=500)
+    arttype = models.ForeignKey("ArtType", on_delete=models.SET_NULL, related_name='arttype_pieces')
+    subtype = models.ManyToManyField("SubTypes", through="piecesubtypes", related_name='pieces_with_subtype', related_name='subtype_pieces')
+    media = models.ForeignKey("Media", on_delete=models.SET_NULL, related_name='pieces_using_this_media')
+    length = models.IntegerField(max_length=3)
+    width = models.IntegerField(max_length=3)
+    height = models.IntegerField(max_length=3)
+    weight = models.FloatField(null=True, blank=True)
+    image_url = models.CharField(max_length=250)
+    about = models.CharField(max_length=1000)
+    available_purchase = models.BooleanField(default=False)
+    available_show = models.BooleanField(default=False)
+    will_ship = models.BooleanField(default=False)
+    unique = models.BooleanField(default=False)
+    quantity_available = models.IntegerField(max_length=5)
```

Comment:
Would be nice to add a validator where qty available is only able to be over 0 if unique is False.

Reply:
Will add to stretch tickets